numbers/integrate.go

```diff
@@ -45,6 +45,7 @@ func LogIntegrateIterative(f func(float64) float64, a float64, b float64, maxIte
 		n := n * 10
 		curr := LogIntegrate(f, a, b, n)
 		if (prev-curr)/curr < relativeError {
```

Comment:
I am not sure that prev will be greater than curr, so I think this numerator needs to be wrapped in math.Abs

Reply:
Great catch!